lib/src/model/hms_ios_pip_controller.dart

```diff
@@ -0,0 +1,187 @@
+import 'package:flutter/material.dart';
+import 'package:hmssdk_flutter/hmssdk_flutter.dart';
+import 'package:hmssdk_flutter/src/service/platform_service.dart';
+
+/// [HMSIOSPIPController] is used to setup and start the PIP in iOS. To know more visit [here](https://www.100ms.live/docs/flutter/v2/advanced-features/pip-mode).
+class HMSIOSPIPController {
+  static bool _isPIPSetupDone = false;
+
+  ///setup PIP is used to setup PIP in iOS devices.
```

Comment:
Add comments about possible points where setup PIP function can be called like - onJoin, or on click of a button, or when a Screenshare starts, etc